### DISCUSSION TITLE
How do I configure VERCEL_ANALYTICS_ID in Sveltekit when using the Vercel CLI --prebuilt

### DISCUSSION THREAD

**Discussion by alexbjorlig** [vercel/community#3852] [Help]
### Summary

My goal is to setup Sveltekit with the speed insights API.

However, I'm unsure how to configure the VERCEL_ANALYTICS_ID, because I'm building and deploying from a Github Action (using `vercel deploy --prebuilt`)

What are my options?

### Steps to Reproduce

Setup a github action and deploy a Sveltekit project with `vercel deploy --prebuilt`

**Comment by amyegan** [comment]
Hey @alexbjorlig. I just tried this from local with the [SvelteKit starter repo](https://github.com/vercel/vercel/tree/main/examples/sveltekit), which includes the code for Speed Insights. When I pull the environment variables for my build, I see `VERCEL_ANALYTICS_ID` appear in my project's `.vercel/.env.production.local` file. My prebuilt deployment has the correct value (I logged it to be sure) and Speed Insights data appeared in my dashboard after I enabled the feature.

Please let me know if you run into any problems with this 🙏

**Reply by alexbjorlig** [reply]
Hmm thanks for testing that. The thing is that we build our project with [Doppler](https://www.doppler.com/), so we don't actually "pull" the environment variables.

Is there a command with the CLI where I can get the `VERCEL_ANALYTICS_ID` - or is there some other way of getting it, so I can just save it in Doppler?

**Reply by alexbjorlig** [reply]
I would really like to enable this feature 🤗
